packages/pure-markdown/src/index.ts

```diff
@@ -4,6 +4,8 @@
  * Note that this file may be used in stand alone nodejs, thus
  * do not import anything from Perseus
  */
+export {libVersion} from "./version";
```

Comment:
**suggestion (non-blocking):** `version` feels like it's more common than `libVersion`.

Reply:
The only reason I like `libVersion` is that in Perseus, specifically, there are several "version" exports. With `@khanacademy/perseus` exporting `apiVersion`, `itemDataVersion`, and `itemVersion`, I thought `libVersion` was more clear than simply `version`.

<img width="526" alt="image" src="https://github.com/Khan/perseus/assets/77138/75fef3b1-a1aa-44fe-82c9-7928aae27386">